contracts/child/ChildToken.sol

```diff
@@ -23,9 +19,34 @@ contract ChildToken is Ownable {
   }
 
   function deposit(address user, uint256 amountOrTokenId) public;
-
   function withdraw(uint256 amountOrTokenId) public;
-
   function setParent(address _parent) public;
 
+  function ecrecovery(
```

Comment:
we should use this from lib only?

Reply:
Yes, should have used the lib `ECVerify` that contains this code but there was a problem in migrations since we were deploying this lib on both main and child chain. So I put this here.